flow/Arena.cpp

```diff
@@ -454,7 +454,7 @@ void testIteratorIncrement() {
 			i += 2;
```

Comment:
I guess that means that these other places where we increment an iterator and leave it in an un-dereferenceable state (but also not equal to `end()`) are also illegitimate. Would you mind changing all the += and -= 2s to 1s?

Reply:
Done.